osu.Game.Tests/Beatmaps/IO/LineBufferedReaderTest.cs

```diff
@@ -113,19 +113,14 @@ public void TestReadToEndNoPeeks()
         [Test]
         public void TestReadToEndAfterReadsAndPeeks()
         {
-            const string contents = @"this line is gone
-this one shouldn't be
-these ones
-definitely not";
+            const string contents = "this line is gone\rthis one shouldn't be\r\nthese ones\ndefinitely not";
 
             using (var stream = new MemoryStream(Encoding.UTF8.GetBytes(contents)))
             using (var bufferedReader = new LineBufferedReader(stream))
             {
                 Assert.AreEqual("this line is gone", bufferedReader.ReadLine());
                 Assert.AreEqual("this one shouldn't be", bufferedReader.PeekLine());
-                const string ending = @"this one shouldn't be
-these ones
-definitely not";
+                const string ending = "this one shouldn't be\nthese ones\ndefinitely not";
```

Comment:
Needs investigation as to why `ReadToEnd()` is providing a different output to the input above.

Reply:
@bdach if you beat me to it, wanna take a look into this? not sure if it's a regression in .NET core 3.0 or an expected change (seems that `\r\n` are being replaced with `\n` on macOS, at least).

Reply:
Sure, I'll have a look at it as the first thing today.

Reply:
Looks like this is also reproducing on master, so may have just never been run/tested on a non-windows host.